### PR TITLE
Fixed bug where stored claims could be minipulated

### PR DESCRIPTION
**Pull request by blackstardlb**
Fixed bug where stored claims could be manipulated because EphemeralConnector cache and EphemeralStorage would not store and retrieve deep copies.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,23 +25,24 @@
     "@discipl/core-baseconnector": "0.2.2",
     "express": "4.17.1",
     "json-stable-stringify": "1.0.1",
+    "lodash": "4.17.20",
```
Comment (pimotte) [inline package.json:28]: With respect to dependencies, lodash is a little bit heavier than I'm comfortable pulling in. If we have a smaller stable library that provides this functionality, I'd be okay with pulling that in (like json-stable-stringify). Alternatively, we could include our own utility function.

**Review by pimotte — CHANGES_REQUESTED** [review]
Solid work, nice catch on that it can be altered after claiming as well, I don't think I would have gotten that.

Only point to discuss is the inclusion of lodash

File: package.json (hunk 1 above)
Comment (blackstardlb) [reply package.json]: The Lodash.cloneDeep implementation seems pretty complex. I found a package that allowed the import of only that function.

**Review by pimotte — APPROVED** [review]
:partying_face:

**Pull request merged**